bluemix/env_test.go

```diff
@@ -9,8 +9,6 @@ import (
 )
 
 func TestGet(t *testing.T) {
-	assert.Empty(t, EnvTrace.Get())
```

Comment:
I'm curious why this was necessary. Is this unrelated to the new addition? Were the tests in general failing on this check for Windows?

Reply:
The test will fail if IBMCLOUD_TRACE is set to any value on the computer running the trace. I usually have it set so it was a bit annoying for me personally.  `EnvTrace.Get()` is tested later on in the function so I didn't feel like adding an `os.Unsetenv` here would have been very useful so I just removed this assert.